### PR TITLE
First version adding lock

### PR DESCRIPTION
**Pull request by ricardoul**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "test": "jest",
     "test:coverage": "jest --coverage --forceExit",
-    "migrate": "migrate",
+    "migrate": "node ./lock.js",
```
Comment (damsal7) [inline package.json]: and what about the commands "migrate:up" and "migrate:down"?

File: lock.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
+var Redlock = require('redlock');
```
Comment (damsal7) [inline lock.js]: My only concern here is the code format

File: package.json (hunk 1 above)
Comment (ricardoul) [reply package.json]: I think migrate:up and migrate:down will be manual commands, so is ok to don't be locked, what do you think?

File: lock.js (hunk 2 above)
Comment (msazdova) [reply lock.js]: I agree with @damsal7 on the code format, but bcx code is different from platform so we're probably not the right people to be asked. :D

Comment (ricardoul) [reply lock.js]: Just run the linter

**Pull request closed without merging**